### ISSUE TITLE
[feature request] Upstream to core PyTorch antialiased interpolation

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#71638]
### 🚀 The feature, motivation and pitch

It was implemented for some cases in torchvision in https://github.com/pytorch/vision/pull/3761, but was not upstreamed as far as I can see, so other users of F.interpolate and not of torchvision.transforms.functional.resize can't take advantage of it

**Comment by vadimkantorov** [comment]
Probably antialiasing could be useful in audio domain as well for simple resampling...

**Comment by jbschlosser** [comment]
Hey @vadimkantorov, I believe anti-aliasing on CPU was added for bilinear interpolation in #65142 and bicubic interpolation in #68819. A CUDA implementation is still under review in #70930. Is there anything missing once that merges?

**Comment by vadimkantorov** [comment]
oh, then I guess torchvision should migrate to pytorch core :) was there anything special done for resampling in torchaudio?

**Comment by jbschlosser** [comment]
Not sure about that, but you could check with the torchaudio team. cc @mthrok

**Comment by vadimkantorov** [comment]
Otherwise, this could probably be closed :) It was just strange to me that torchvision was still using antialias special cases in F.resize ~9 months after they were implemented
